companies/rechargecom.json

```
@@ -0,0 +1,28 @@
+{
+    "slug": "rechargecom",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "telecommunication"
+    ],
+    "name": "Recharge.com International B.V.",
+    "runs": [
+        "recharge.com",
+        "rapido.com",
+        "mobiletopup.co.uk",
+        "recharge.fr",
+        "herladen.com",
+        "guthaben.de",
+        "beltegoed.nl"
+    ],
+    "address": "Amsteldijk 216\n1079 LK Amsterdam\nThe Netherlands",
+    "phone": "+31 88 006 4109",
+    "email": "datarequest@cg.nl",
+    "sources": [
+        "https://www.guthaben.de/datenschutzerklaerung",
+        "https://www.guthaben.de/impressum"
+    ],
+    "suggested-transport-medium": "email",
+    "quality": "verified"
```

```suggestion
    "quality": "tested"
```

I requested my data and they managed to answer surprisingly quickly. The email seems to work fine.

---

You'll need to set `required-elements` as well. And please also set `needs-id-document`, I'm considering making that a requirement as well.